src/main/java/cinema/Main.java

```diff
@@ -44,5 +49,9 @@ public static void main(String[] args) {
         System.out.println("Requested date: " + requestedDate);
         List<MovieSession> res = movieSessionService.findAvailableSessions(1L, requestedDate);
         System.out.println(res);
+
+        User newUser = authService.register("test@gmail.com", "1234");
```

Comment:
what if we create new user within the same email?

Reply:
Weird question, an exception will be thrown

Reply:
I mean it would be nice to handle such things. Up to you

Reply:
main method will throw an exception. Such exception will be handled in other places (controllers for example). Here we can leave it as it is. But you are right, In general, we should handle such situations